src/common/pico_base/include/pico/config.h

```diff
@@ -18,4 +18,9 @@
 
 #include "pico/config_autogen.h"
 
+// PICO_CONFIG: PICO_CONFIG_RTOS_ADAPTER_HEADER, path to header include in the default pico/config.h for RTOS integration defines that must be included in all sources, type=string, default=none, group=pico_base
+#ifdef PICO_CONFIG_RTOS_ADAPTER_HEADER
+#include __PICO_XSTRING(PICO_CONFIG_RTOS_ADAPTER_HEADER)
```

Comment:
Does the use of `__PICO_XSTRING` here mean that `pico/config.h` will _only_ ever be included directly by `pico.h` ?

Reply:
Good point, but I think it is fair. The user (and all our headers) are supposed to include `pico.h` as this includes all the other stuff in the right order.